keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,320 @@
+room for maintaining their mental health, prepare for events like KubeCon or
+work on the downstream integration.
+
+Through this proposal SIG Release's aim is to give contributors more
+flexibility to decide how to invest their time. It is explicitly _not_ to push
+contributors in doing more.
+
+#### SIG Release members
+
+By applying a cadence of 3 releases per year, SIG Release members will gain a
+reduced management overhead. There are also only 3 patch releases to maintain,
+which right now can overlap up to 4. SIG Release will gain more time to ensure
+a seamless transition from the previous release team to the next one. It is
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies — for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+#### Feature graduation
+
+Research discovered that only 5% of Kubernetes features advanced from Alpha to
+GA in the minimum 3 releases. However, the same research showed that reminders
+from the Release Team played a critical role in advancement of more than 50% of
+features. With an increased release cycle, this reminder activity can be
+expected to slow down. As such, advancement will need to be mitigated by making
+sure that SIGs keep track of their feature enhancement in more detail.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
```

Comment:
It seems implicit that we will adopt this schedule at the beginning of next calendar year, i.e. takes effect in 2022. Is that an accurate interpretation and if so, can we make this explicit?

Reply:
The intent is to start with 1.22 I believe, ( so not 2022). Should we clarify that this will start with the 1.22 cycle?

Reply:
In the summary toward the top of the KEP, I added an explicit statement that this will start with the 1.22 release cycle.